### ISSUE TITLE
linuxmuster-prepare with from scratch installation -  quotas on root / activated but do not work

### ISSUE THREAD

**Issue by cweikl** [linuxmuster/linuxmuster-prepare#13]
Using this script on a from scratch installation, it leads to a new /etc/fstab 
setting a quota for / into action.
During startup quotaon fails because the files aquota.group and aquota.user are still missing in /.
In /srv/samba/global these files have been added correctly.
Maybe we do not need any Quota on / ? If so, the above mentioned files should also be generated during installation.
In addtion, there should be a test if a lvm is already present, if so no quotas on / should be implemented. 
The same is true for perpraring a lvm with this script.
I am not sure if this issue is rather for another script / package, but I foiunt this code:

```
def do_fstab_root(quotamntopts):
    mntopts = quotamntopts + ',errors=remount-ro'
    config = FSTabConfig(path='/etc/fstab')
    config.load()
    count = 0
    count_max = len(config.tree.filesystems)
    while count < count_max:
        mp = config.tree.filesystems[count].mountpoint
        #print('## Mountpoint ' + str(count) + ' ' + mp)
        if mp == '/':
            print('# Modifying mount options for "' + mp + '".')
            config.tree.filesystems[count].options = mntopts
            config.save()
            os.system('mount -o remount /')
            break
        count += 1

```
Maybe a command can be integrated to generate the above mentioned files in /
Thanks.

**Comment by HappyBasher** [comment]
linuxmuster-prepare is a tool to prepare a system with a certain standard partition layout for setup. If you prefer a custom partition layout you are free to do this. There is no need to use linuxmuster-prepare in advance to install a linuxmuster.net system.

**Comment by HappyBasher** [comment]
An additional -m parameter to quotacheck should do the trick.

**Comment by HappyBasher** [comment]
Published in linuxmuster-prepare (0.7.6-0ubuntu0). Thank you for your contribution.
